Search WordPress post

xentenza opened the pull request:
Permettre de rechercher un post de blog WordPress (notamment quand ils sont utilisés pour des archives comme Campus Lectures, Archizoom, etc.)

xentenza committed:
Search WordPress post

Permettre de rechercher un post de blog WordPress (notamment quand ils sont utilisés pour des archives comme Campus Lectures, Archizoom, etc.)

williambelle commented on assets/components/molecules/filters/filters-blog.twig:
```diff
@@ -16,6 +16,16 @@
     <form class="p-3 p-lg-0 mt-md-3">
 
+      <div class="form-group">
+        <form class="border-0 p-0" method="get" name="?s+s">
```
Il y a 2 `<form>` imbriqués qui donnent un effet d'ombre sous la barre de recherche avec un mobile.

![Screenshot from 2024-06-20 15-59-09](https://github.com/epfl-si/elements/assets/2843501/9d21e7b5-5217-4ae8-8809-b01a31c4edf4)

Je crois que si tu enlèves ce form, c'est bon.

xentenza replied:
@williambelle J'avais surtout fait un beau mélange entre charte graphique et développement.

Je me suis donc concentré que sur le graphisme sur ce coup-là. La mise en place du script pour effectuer les recherches, ce sera avec les équipes WP 😊

xentenza committed:
Label customization

- added the ability to customize form label names
- added a search field to the blog homepage sidebar
